### DISCUSSION TITLE
Reformatting  "vhost/vhostname" in rabbitmq docs.

### DISCUSSION THREAD

**Discussion by Shubham82** [kedacore/keda#1430] [General] [answered]
In this vhostname and vhosts are same same thing
https://keda.sh/docs/2.0/scalers/rabbitmq-queue/#authentication-parameters
For more robust user experience and symmetry would it be better to use either vhost or vhostname?

**Comment by tomkerkhove** [comment]
Are you sure they are the same thing? I'm not a RabbitMQ expert so just confirming that.

If they are, then we can improve the docs indeed.

**Comment by Shubham82** [comment]
AFAIK they both are same. i also asked on keda slack, here is the screenshot of it
![screenshot](https://user-images.githubusercontent.com/50987400/102327466-c874fe00-3fab-11eb-8c86-b34e59fcade6.png)
if it would sound good i will change it. Maybe @jeffhollan can tell more about it.

**Reply by tomkerkhove** [reply]
@zroubalik Do you know?

**Reply by zroubalik** [reply]
Sorry I am no means a Rabbit MQ expert 🤷‍♂️

**Reply by tomkerkhove** [reply]
Adding @ahmelsayed @yaron2 who implemented this, AFAIK, who might be able to confirm this.

**Reply by zroubalik** [reply]
You might take a look on this new PRs: https://github.com/kedacore/keda/pull/1451 https://github.com/kedacore/keda-docs/pull/339

**Reply by Shubham82** [reply]
I have looked into this PR, In this vhostName added in metadata to override vhost in host string.  and as coderanger suggested these are just two formatting example as also mentioned in the screenshot. so i would suggest  to go for vhost for better understanding rather than vhostname because vhostname now used in metadata.

**Comment by Shubham82** [comment, marked as answer]
Hi!
Any update on this!

**Reply by tomkerkhove** [reply]
This is merged: https://github.com/kedacore/keda-docs/pull/339

**Reply by Shubham82** [reply]
Hi @tomkerkhove
i see this, but in this there is no change in authentication-parameters(https://github.com/kedacore/keda-docs/blob/master/content/docs/2.1/scalers/rabbitmq-queue.md#authentication-parameters),  this is just two formatting examples, is this ok if we would use one of them for better user experience?
Thanks

**Reply by tomkerkhove** [reply]
I'm pulling in @coderanger who is more knowledgable on RabbitMQ than @zroubalik or myself.
If you think it's best to align the names and that's the common standard in RabbitMQ world then that's fine for me.

**Reply by Shubham82** [reply]
Thanks @tomkerkhove i think it should be same (vhost). will wait for @coderanger confirmation. After that we will close this.

**Reply by coderanger** [reply]
Yeah they should definitely be the same, though which it is kind of doesn't matter. We use `queueName` and `vhostName` as the parameters but `/vhost` would be the more expected thing in a URI I think.

**Reply by Shubham82** [reply]
thanks @coderanger for confirming this, that is also my concern. @tomkerkhove  if its sound good then  should i raise PR for it?

**Reply by tomkerkhove** [reply]
Definately, thanks!
